source/index.html

```diff
@@ -4,9 +4,252 @@
+            от использования приложения длится целые сутки!
+          </p>
+          <a href="#">Узнать больше об исследовании</a>
+        </div>
+
+        <div class="reviews__wrapper slider">
+          <h2 class="visually-hidden">Отзывы</h2>
+          <ul class="reviews__list slider__list">
+            <li class="reviews__item reviews__item--first slider__item">
+              <h3 class="reviews__autor-name">Николай Петров</h3>
+              <p class="reviews__description">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </li>
+            <li class="reviews__item reviews__item--second slider__item">
+              <h3 class="reviews__autor-name">Николай Петров</h3>
+              <p class="reviews__description">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </li>
+            <li class="reviews__item reviews__item--third slider__item">
+              <h3 class="reviews__autor-name">Николай Петров</h3>
+              <p class="reviews__description">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </li>
+          </ul>
+          <p class="reviews__toggles slider__toggles">
+            <button class="slider__toggle slider__toggle-current" type="button">1</button>
+            <button class="slider__toggle" type="button">2</button>
+            <button class="slider__toggle" type="button">3</button>
+          </p>
+        </div>
+      </section>
+
+      <section class="price">
+        <div class="price__wrapper">
+          <h2 class="visually-hidden">Ценник</h2>
+          <table class="price__list">
+            <thead>
+              <tr>
+                <th>СТАНДАРТ<br> 3,99 USD</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Розовый фильтр</td>
+              </tr>
+              <tr>
+                <td>Смайлики</td>
+              </tr>
+              <tr>
+                <td>Комментарии</td>
+              </tr>
+            </tbody>
+          </table>
+          <p class="price__toggles slider__toggles">
+            <button class="slider__toggle" type="button">1</button>
+            <button class="slider__toggle slider__toggle-current" type="button">2</button>
+            <button class="slider__toggle" type="button">3</button>
+          </p>
+        </div>
+      </section>
+
+      <section class="contacts">
+        <h2 class="visually-hidden">Конатакты</h2>
+        <div class="contact__wrapper">
```

Comment:
```suggestion
        <div class="contacts__wrapper">
```

блок называется `contacts`. Нужно подправить и другие элементы